common/lib/xmodule/xmodule/partitions/partitions_service.py

```diff
@@ -3,41 +3,114 @@
 user partitions.  It uses the user_service key/value store provided by the LMS runtime to
 persist the assignments.
 """
-from abc import ABCMeta, abstractproperty
+from django.conf import settings
```

Comment:
@nasthagiri are these django imports OK in this file? Other files in common/lib/module use this workaround instead of importing django.utils.translation:

````
# Make '_' a no-op so we can scrape strings. Using lambda instead of
#  `django.utils.translation.ugettext_noop` because Django cannot be imported in this file
_ = lambda text: text